packages/mongodb-rag-core/src/contentStore/MongoDbPageStore.test.ts

```diff
@@ -221,4 +221,52 @@ describe("MongoDbPageStore", () => {
       "custom-pages"
     );
   });
+
+  describe("deletePages", () => {
+    it("marks all pages as deleted", async () => {
+      assert(store);
+      await store.updatePages(moviePages);
+      let pages = await store.loadPages();
+      expect(pages.length).toBe(5);
```

Comment:
can you do this logic in a beforeEach block? it's repeated in each test. redundant

Reply:
can also add an afterEach block to delete all pages in collection

Reply:
I added a beforeEach block, but this is nested in a describe block with an afterEach block that is already dropping the db and closing the connection. It feels weird to use store.deletePages in the hook when thats the function the test is testing

Reply:
>I added a beforeEach block, but this is nested in a describe block with an afterEach block that is already dropping the db and closing the connection.

should be fine

> t feels weird to use store.deletePages in the hook when thats the function the test is testing

agreed